consensus/replay.go

```diff
@@ -320,6 +326,7 @@ func (h *Handshaker) replayBlocks(proxyApp proxy.AppConns, appBlockHeight, store
 func (h *Handshaker) replayLastBlock(proxyApp proxy.AppConnConsensus) ([]byte, error) {
 	mempool := types.MockMempool{}
 	cs := NewConsensusState(h.config, h.state, proxyApp, h.store, mempool)
+	defer cs.Stop()
```

Comment:
is there a particular reason to place `defer cs.Stop` before `cs.Start`? I thought the common pattern in Golang is `smth.Start(); defer smth.Stop()`